### ISSUE TITLE
Example code not matching finished program

### ISSUE THREAD

**Issue by spectrumcomputing** [dotnet/docs#17384]
Firstly, thank you for spending the time creating such great tutorials.  I especially like the new C# 101 videos.  Hopefully you will produce more?

The section on 'Search and Sort Lists' shows:

```
var index = names.IndexOf("Felipe");
if (index == -1)
{
    Console.WriteLine($"When an item is not found, IndexOf returns {index}");
}
else
{
    Console.WriteLine($"The name {names[index]} is at index {index}");
}

index = names.IndexOf("Not Found");
if (index == -1)
{
    Console.WriteLine($"When an item is not found, IndexOf returns {index}");
}
else
{
    Console.WriteLine($"The name {names[index]} is at index {index}");

}
```
But the finished example further down the page shows it as:

```
 var index = names.IndexOf("Felipe");
            Console.WriteLine($"The name {names[index]} is at index {index}");

            var notFound = names.IndexOf("Not Found");
            Console.WriteLine($"When an item is not found, IndexOf returns {notFound}");

            names.Sort();
            foreach (var name in names)
            {
                Console.WriteLine($"Hello {name.ToUpper()}!");
            }
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ef7201b-e3d3-484c-c46f-0f2f257ddae3
* Version Independent ID: bc2b573a-40ac-a9fd-daa8-b5a5884b11c7
* Content: [Work with collections - Introduction to C# tutorial](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/arrays-and-collections#feedback)
* Content Source: [docs/csharp/tutorials/intro-to-csharp/arrays-and-collections.md](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/intro-to-csharp/arrays-and-collections.md)
* Product: **dotnet-csharp**
* Technology: **csharp-get-started**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Thanks for writing this @spectrumcomputing 

The first example is the correct code. I've added this to our backlog, and added the "up for grabs" label.

We'll address it when we next update this article, or community members can submit a fix in the interim, if interested.

**Comment by spectrumcomputing** [comment]
Thanks for the reply @BillWagner 

This is my completed code

```
using System;
using System.Collections.Generic;

namespace list_tutorial
{
    class Program
    {
        static void Main(string[] args)
        {
            var names = new List<string> { "<name>", "Ana", "Felipe" };
            foreach (var name in names)
            {
                Console.WriteLine($"Hello {name.ToUpper()}!");
            }

            Console.WriteLine();
            names.Add("Maria");
            names.Add("Bill");
            names.Remove("Ana");
            foreach (var name in names)
            {
                Console.WriteLine($"Hello {name.ToUpper()}!");
            }

            Console.WriteLine($"My name is {names[0]}");
            Console.WriteLine($"I've added {names[2]} and {names[3]} to the list");

            Console.WriteLine($"The list has {names.Count} people in it");

            var index = names.IndexOf("Felipe");
            if (index == -1)
            {
                Console.WriteLine($"When an item is not found, IndexOf returns {index}");
            }
            else
            {
                Console.WriteLine($"The name {names[index]} is at index {index}");
            }

            index = names.IndexOf("Not Found");
            if (index == -1)
            {
                Console.WriteLine($"When an item is not found, IndexOf returns {index}");
            }
            else
            {
                Console.WriteLine($"The name {names[index]} is at index {index}");

            }

            names.Sort();
            foreach (var name in names)
            {
                Console.WriteLine($"Hello {name.ToUpper()}!");
            }
        }
    }
}
```

**Comment by BillWagner** [comment]
Fixed in #17395
